src/QueryBuilder/helpers.js

```diff
@@ -0,0 +1,23 @@
+export const getTableMetadata = (entityType) => {
```

Comment:
if it's Viewer specific util, please move it to QueryViewer

Reply:
done